### PR TITLE
#33 - replace etcd based dogu registry with configMap based one

### PR DESCRIPTION
**Pull request by meiserloh**

### CODE REVIEW

File: docs/development/development_de.md
Comment (jelemux) [inline docs/development/development_de.md:1]: Why was this deleted?

File: packages/debug/doguLogLevelRegistry.go [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
+		doguReg:             doguReg,
+		logLevelRegistryMap: map[string]string{},
 	}
 }
 
 // MarshalFromCesRegistryToString marshals the registry to yaml string.
-func (d *doguLogLevelYamlRegistryMap) MarshalFromCesRegistryToString(cesRegistry cesRegistry) (string, error) {
-	d.registry = map[string]string{}
+func (d *doguLogLevelYamlRegistryMap) MarshalFromCesRegistryToString(ctx context.Context) (string, error) {
+	d.logLevelRegistryMap = map[string]string{}
```
Comment (jelemux) [inline packages/debug/doguLogLevelRegistry.go:30]: Isn't this already initialized by the constructor?

File: packages/debug/doguLogLevelRegistry.go [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
 	return &doguLogLevelYamlRegistryMap{
-		registry: map[string]string{},
+		cesRegistry:         cesRegistry,
+		doguReg:             doguReg,
+		logLevelRegistryMap: map[string]string{},
 	}
 }
 
 // MarshalFromCesRegistryToString marshals the registry to yaml string.
-func (d *doguLogLevelYamlRegistryMap) MarshalFromCesRegistryToString(cesRegistry cesRegistry) (string, error) {
```
Comment (jelemux) [inline packages/debug/doguLogLevelRegistry.go:23]: I like that you moved the `cesRegistry` into the struct.

File: packages/doguAdministration/service.go [hunk 3]
```diff
@@ -93,44 +91,22 @@ func getGRPCInternalDoguActionError(verb string, err error) error {
...
+	return s.createDoguListResponse(ctx, doguJsonList), nil
 }
 
-func (s *server) createDoguListResponse(dogus []*core.Dogu) *pb.DoguListResponse {
+func (s *server) createDoguListResponse(ctx context.Context, dogus []*core.Dogu) *pb.DoguListResponse {
 	var result []*pb.Dogu
 
 	for _, dogu := range dogus {
 
-		logLevel, err := s.loggingService.GetLogLevel(context.TODO(), dogu.GetSimpleName())
```
Comment (jelemux) [inline packages/doguAdministration/service.go:133]: Good catch!

**Review by jelemux — COMMENTED** [review]
Two questions, looks good otherwise!

File: docs/development/development_de.md
Comment (meiserloh) [reply docs/development/development_de.md:1]: all grpc endpoints are not part of this repo anymore, so this part of the documentation is not necessary and does not work anymyore.

https://github.com/cloudogu/ces-control-api

File: packages/debug/doguLogLevelRegistry.go (hunk 1 above)
Comment (meiserloh) [reply packages/debug/doguLogLevelRegistry.go]: that's correct ^^'

**Review by jelemux — APPROVED** [review]
LGTM!
